checkov/sca_image/runner.py

```diff
@@ -183,6 +182,26 @@ def iterate_image_files(self, abs_fname: str, report: Report, runner_filter: Run
                                                          runner_filter=runner_filter)
                     merge_reports(report, image_report)
 
+    def get_report_from_scan_result(self, result: Dict[str, Any], dockerfile_path: str, rootless_file_path: str,
+                                    image_details: ImageDetails | None, runner_filter: RunnerFilter) -> Report:
+        report = self.get_empty_report()
+        vulnerabilities = result.get("vulnerabilities") or []
+        packages = result.get("packages") or []
```

Comment:
```suggestion
        vulnerabilities = result.get("vulnerabilities", [])
        packages = result.get("packages", [])
```

Reply:
@matansha I agree. not sure why, but this is how it happens in the rest of the code, but let's fix as you suggested